libvast/vast/fwd.hpp

```diff
@@ -91,6 +91,9 @@ void intrusive_ptr_add_ref(const table_slice*);
 void intrusive_ptr_release(const table_slice*);
 table_slice* intrusive_cow_ptr_unshare(table_slice*&);
 
+void intrusive_ptr_add_ref(const table_slice_builder*);
+void intrusive_ptr_release(const table_slice_builder*);
```

Comment:
Just out of curiosity: why are these needed?

Reply:
Are you asking why these functions exist in the first place or why are they forward declared here?

Reply:
The former.